modules/covid_vaccine/app/controllers/covid_vaccine/v0/expanded_registration_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require_relative '../../../serializers/covid_vaccine/v0/expanded_registration_serializer'
+
+module CovidVaccine
+  module V0
+    class ExpandedRegistrationController < CovidVaccine::ApplicationController
+      skip_before_action :validate_session
+      before_action :validate_raw_form_data, only: :create
+      wrap_parameters :registration
+
+      def create
+        raw_form_data = params[:registration]
+        record = CovidVaccine::V0::ExpandedRegistrationSubmission.create!({ submission_uuid: SecureRandom.uuid,
+                                                                            raw_form_data: raw_form_data })
+
+        formatted_date = record.created_at.strftime('%B %-d, %Y %-l:%M %P %Z').sub(/([ap])m/, '\1.m.')
+        CovidVaccine::ExpandedRegistrationEmailJob.perform_async(record.id,
+                                                                 raw_form_data[:email],
+                                                                 formatted_date)
```

Comment:
These attributes: `raw_form_data`, `created_at` should be available from the record. Why wouldn't we just load it from sidekiq? Is this just a performance tweak because raw_form_data is encrypted?

Any reason formatted_date couldn't live as a method on the job?

Reply:
fair point, this implementation was path-dependent on using the other job as a starting point. I'll refactor so the only argument is the record_id.